Taskfile.yml

```diff
@@ -110,10 +110,14 @@ tasks:
     desc: Run the Go-based integration tests
     deps:
       - task: go:build
+        vars:
+          EXTRA_FLAGS: "-covermode=atomic"
     dir: '{{default "./" .GO_MODULE_PATH}}'
     cmds:
       - |
-        go test \
+        rm -fr coverage_data
+        mkdir coverage_data
+        INTEGRATION_GOCOVERDIR={{ .ROOT_DIR }}/coverage_data go test \
```

Comment:
Shouldn't be possible to avoid this and use directly `GOCOVERDIR` env var instead? Without modifying the `arduino-cli.go`. See `Running a coverage-instrumented binary` section [here](Running a coverage-instrumented binary)

Reply:
No, I tried it, as I've explained in the PR comment:
>  1. To produce the coverage output, the instrumented executable must be run with a GOCOVERDIR environment variable set to an existing directory path where the coverage data will be written. To accomplish this, I've added to the integration test the support for another env var INTEGRATION_GOCOVERDIR, the content of this variable will be passed to the arduino-cli run as GOCOVERDIR. **We can not use GOCOVERDIR directly because it is overwritten by the `go test ...` command.**